src/main/java/sg/ncl/MainController.java

```diff
@@ -2413,11 +2416,11 @@ private User2 extractUserInfo(String userJson) {
         user2.setStatus(object.getString("status"));
         user2.setEmailVerified(object.getBoolean("emailVerified"));
 
-        String role = UserType.USER.toString();
-        if (object.getJSONArray("roles") != null) {
-            role = object.getJSONArray("roles").get(0).toString();
-        }
-        user2.setRoles(role);
+//        String role = UserType.USER.toString();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)